src/panels/lovelace/cards/hui-picture-elements-card.js

```diff
@@ -55,14 +26,7 @@ class HuiPictureElementsCard extends NavigateMixin(EventsMixin(LocalizeMixin(Pol
         padding: 8px;
         white-space: nowrap;
       }
-      .clickable {
-        cursor: pointer;
-      }
-      ha-call-service-button {
-        color: var(--primary-color);
-        white-space: nowrap;
-      }
-      hui-image {
+      hui-image-element {
```

Comment:
What's with the state label padding and white space CSS above this line? Shouldn't that be put inside the web compoennt.

Reply:
I'll move it. I missed adding the class to the div anyway. I just thought we wanted to avoid applying specific rules within components that will be reused in multiple places

Reply:
Think of each web component (all of them) as an individual web component, not related to the current context that it's being rendered in. I should be able to take any of these element components and use them anywhere in the frontend, without requiring the rendering component to know _any_ implementation detail like having to add CSS.